x11-packages/rnote/build.sh

```diff
@@ -0,0 +1,67 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/flxzt/rnote"
+TERMUX_PKG_DESCRIPTION="An infinite canvas vector-based drawing application for handwritten notes"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.11.0"
+TERMUX_PKG_SRCURL="https://github.com/flxzt/rnote/archive/v${TERMUX_PKG_VERSION}/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256=b133d4331963d3c09d3a7477f60fc4c5072471dcbf459379a593ca1724164af4
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="gdk-pixbuf, gettext, glib, graphene, gtk4, hicolor-icon-theme, libadwaita, libcairo, pipewire, pango, poppler"
+TERMUX_PKG_BUILD_DEPENDS="libiconv"
+
+__fetch_gettext_rs() {
+    # This version is relative to gettext-sys provided by the gettext-rs crate
```

Comment:
Gonna need tabs here as well.
```suggestion
	# This version is relative to gettext-sys provided by the gettext-rs crate
```